src/Main.java

```diff
@@ -1,5 +1,50 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        System.out.println("Задача 1.");
+        int[] arr1 = {10000, 10500, 12300, 23400, 5000};
+        int sum = 0;
+        for (int index = 0; index < arr1.length; index++) {
+            sum += arr1[index];
+        }
+        System.out.println("Сумма трат за месяц составила " + sum + " рублей");
+        System.out.println("Задача 2.");
+        int[] arr2 = {10000, 10500, 12300, 23400, 5000};
+        int max = -1;
```

Comment:
Это работает в конкретном примере, но если в массиве будут только отрицательные числа меньше -1, то код перестанет корректно работать. Я рекомендую приравнивать к значению первого элемента массива
```suggestion
        int max =  arr2[0];
```